backend/route/views.py

```diff
@@ -176,6 +176,26 @@ def post_spec_get(request, ID):
         }
     return JsonResponse(response_dict, safe=False)
     
+@require_POST
+def post_share(request, post_id):
+    logged_user_id=request.session.get('user', None)
+
+    try:
```

Comment:
You can use the `get_object_or_404()` method from `django.shortcuts` in this case.
It is the same with `Model.objects.get()` method but automatically returns a 404 response if it fails to get an object.

It is just a shortcut, so you can use whatever you want.

Here is the reference:
https://docs.djangoproject.com/en/3.2/topics/http/shortcuts/#get-object-or-404

Reply:
Oh I got it! It looks much better. Thank you for the information!